README.md

```diff
@@ -11,7 +11,8 @@ The application simply declares the data it needs and the Fresh APIs ensure that
 
 ## Caveat
 
-Fresh Data is new. Very new. As such it should *not* be used in production just yet!
+Fresh Data is new. Very new. As such it should *not* be used in production just yet! This code will be changing still.
```

Comment:
So should we use this in `wc-admin`?

Reply:
:trollface:

Reply:
Haha, well, `wc-admin` isn't production yet, is it? :trollface: